### PR TITLE
SAM-3130 [Closed,Open) Intervals in Statistics

### PR DESCRIPTION
**Pull request by alejandrogj**

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 1]
```diff
@@ -2776,81 +2782,38 @@ private int calCorrect(String[] answers, String correct)
...
-      ranges[0] = num + " - " + num;
+      ranges[0] = "[ " + num + " , " + num + " ]";
     }
     else
     {	
-      ranges[0] = (int) min + " - " + (int) (min + interval);
+      ranges[0] = "[ " +(int) min + " , " + (int) (min + interval) + " )";
       int i = 1;
       Integer nextVal = null;
       while(i < ranges.length)
```
Comment (ottenhoff) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java]: Can simplify this while into a for statement?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 2]
```diff
@@ -2752,22 +2752,28 @@ private int calCorrect(String[] answers, String correct)
...
-          }
+            double lowerEndpoint = min + (j * interval);
+            double uppperEndpoint =  min + ((j + 1) * interval);
+
+            if ( j < (numStudents.length -1) ){
+              if( (scores[i] >= lowerEndpoint ) && (scores[i] < uppperEndpoint) ){
+                  numStudents[j]++;
+                  break;
+              }
+           }else{
```
Comment (ottenhoff) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java]: Can you check spacing/tabs here?

Comment (alejandrogj) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java]: I´ve changed the spaces/tabs in this method

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java (hunk 1 above)
Comment (alejandrogj) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java]: "for" statement added

**Pull request merged**